src/Symfony/Component/Security/Http/Authenticator/RememberMeAuthenticator.php

```diff
@@ -57,32 +65,37 @@ public function supports(Request $request): ?bool
             return false;
         }
 
-        // if the attribute is set, this is a lazy firewall. The previous
-        // support call already indicated support, so return null and avoid
-        // recreating the cookie
-        if ($request->attributes->has('_remember_me_token')) {
-            return null;
+        if (($cookie = $request->attributes->get(ResponseListener::COOKIE_ATTR_NAME)) && null === $cookie->getValue()) {
+            return false;
```

Comment:
What condition is this catching? Is it where, earlier in this request, `clearRememberMeCookie()` was called?

Reply:
Yes, e.g. when the user is deauthenticated upon refresh, they should not be authenticated again by the remember me system. See the (fixed) bug reported in https://github.com/symfony/symfony/issues/35198